keyboards/takashicompany/worm_eaten/integrated/rules.mk

```
@@ -0,0 +1,6 @@
+SPLIT_KEYBOARD = no
+INTEGRATED = yes
+
+ifeq ($(strip $(INTEGRATED)), yes)
+    OPT_DEFS += -DINTEGRATED
+endif
```

This is what lint is complaining about, in part.  mostly, you shouldn't need all of this. 
```suggestion
SPLIT_KEYBOARD = no
```